kolibri/plugins/coach/assets/src/views/reports/ReportsQuizLearnerListPage.vue

```diff
@@ -125,16 +123,35 @@
     beforeMount() {
       this.filter = this.filterOptions[0];
     },
+    methods: {
+      completedQuestionsCountLabel(answered, total) {
+        if (answered === total) {
+          return this.$tr('allQuestionsAnswered');
+        } else {
+          return this.$tr('questionsCompletedRatioLabel', { count: answered || 0, total: total });
+        }
+      },
+    },
     $trs: {
       allQuizzes: 'All quizzes',
       activeQuizzes: 'Active quizzes',
       inactiveQuizzes: 'Inactive quizzes',
+      allQuestionsAnswered: 'All questions answered',
       questionsCompletedRatioLabel:
-        '{count, number, integer} of {total, number, integer} {count, plural, other {answered}}',
+        '{count, number, integer} of {total, number, integer} questions {count, plural, other {answered}}',
     },
   };
 
 </script>
 
 
-<style lang="scss" scoped></style>
+<style lang="scss" scoped>
+
+  .small-answered-count {
+    display: block;
+    margin-left: 1.75rem;
```

Comment:
spacing in `8px` increments

Reply:
This is the closest that I can get to the designs using 8px increments (24px).
![8px-increment](https://user-images.githubusercontent.com/6356129/68052353-b4d2df00-fca6-11e9-9dd1-41e94026bb39.png)

The design calls for the text to align with the label in the labeled icon above it:
![design-increments](https://user-images.githubusercontent.com/6356129/68052450-e5b31400-fca6-11e9-80e7-e3b593591186.png)

The px value for this would be 28px (3.5 x 8px).

Should we aim to instead ensure that KLabeledIcon spaces itself in 8px increments? The icon is 15.75px in my browser - then the margin is in `em` https://github.com/learningequality/kolibri/blob/11ae8d25a104945c88c65706ab830da26022973c/packages/kolibri-components/src/KLabeledIcon.vue#L82 - but if that was spaced itself in 8px increments, then spacing the subtext using them would probably be acheivable

Reply:
Got it, thanks for the info! My original note was incorrect.

some additional context:

* The `8px` baseline grid is a rule of thumb, not a requirement
* In the case of `KLabeledIcon`, we used `em` rather than `px` so that the component can be used with different font sizes, e.g. it works in both `<p>` and `<h1>`

Since you're styling the text to match the behavior of `KLabeledIcon`, your original strategy is correct.

Would you mind adding comments to the CSS flagging why these two places use `em`s?

E.g. `ReportsQuizLearnerListPage.vue`:

```css
  .small-answered-count {
    display: block;
    margin-left: 1.75rem; /* matches KLabeledIcon */
```

and `KLabeledIcon.vue`

```css
  .label {
    display: block;
    margin-left: 1.925em; /* scale with parent font size */
  }
```